cluster/ci/config/prow/plugins.yaml

```diff
@@ -1187,6 +1187,12 @@ config_updater:
     cluster/ci/config/mirroring/toolchain/toolchain_*:
       name: image-mirror
 
+    cluster/ci/config/mirroring/codeready-toolchain/host_*:
+      name: image-mirror
+
+    cluster/ci/config/mirroring/codeready-toolchain/member_*:
+      name: image-mirror
+
```

Comment:
Unfortunately this needs to merge separately first, otherwise the bot won't trigger on **this** PR :(

Reply:
ok, let me remove this then. I will add it in another pr

Reply:
removed in c00e173